features/admin_features/administer_restaurants.feature

```diff
@@ -22,9 +22,11 @@ Feature: Admin can administer restaurants
     And I fill in "Name" with "The New Thai Restaurant"
     And I fill in "Address" with "Street 1"
     And I fill in "Description" with "I really cant come up with some boring description"
+    And I add image "restaurant.jpeg"
     And I select "Thai" from restaurant category dropdown
     Then I click "Create Restaurant"
     And I should see "Restaurant was successfully created."
+    And I should see "restaurant.jpeg" image
```

Comment:
I think we need to assert that the other data is also shown on the page, not only the image.

Reply:
We fixed that. @tochman